packages/next/src/lib/worker.ts

```diff
@@ -47,20 +43,13 @@ export class Worker {
     })
 
     const createWorker = () => {
-      // Get the node options without inspect and also remove the
```

Comment:
Maybe we could add some comments that we want to keep the default env here without modification, if any new workers require customization on the NODE_OPTIONS or any env, they should do it on the call site